esphome/components/esp32_ble_client/ble_client_base.h

```diff
@@ -71,6 +72,7 @@ class BLEClientBase : public espbt::ESPBTClient, public Component {
   void set_remote_addr_type(esp_ble_addr_type_t address_type) { this->remote_addr_type_ = address_type; }
   uint16_t get_conn_id() const { return this->conn_id_; }
   uint64_t get_address() const { return this->address_; }
+  bool get_paired() const { return this->paired_; }
```

Comment:
Perhaps a better name?
```suggestion
  bool is_paired() const { return this->paired_; }
```